Production/govuk_ios/ViewControllers/RecentActivityListViewController.swift

```diff
@@ -184,8 +183,11 @@ class RecentActivityListViewController: BaseViewController,
             snapshot.appendItems($0.items, toSection: $0)
         }
         dataSource.apply(snapshot, animatingDifferences: true)
-        tableView.isHidden = viewModel.structure.isEmpty
-        noItemsView.isHidden = !viewModel.structure.isEmpty
+        let hasRecentActivity = !viewModel.structure.isEmpty
+        tableView.isHidden = !hasRecentActivity
+        noItemsView.isHidden = hasRecentActivity
+        let rightNavBarButton = hasRecentActivity ? editButtonItem : nil
+        navigationItem.setRightBarButton(rightNavBarButton, animated: true)
```

Comment:
Alternatively I could only call `setRightBarButton()` when `rightNavBarButton` has actually changed value rather than calling it every time it's enclosing function `reloadSnapshot()` is called?